requirements.txt

```diff
@@ -1,6 +1,6 @@
 joblib
 numpy
 pandas
-scikit-learn>=0.22.0
+scikit-learn>=0.24.0
```

Comment:
Not about this PR, but I am very curious how >= 0.24 will play with other libraries. It would be amazing if we can make a jump to the newer version, across the board.